### ISSUE TITLE
Fediverse addresses are interpreted incorrectly

### ISSUE THREAD

**Issue by towerberlin** [Hypercontext/linkifyjs#476]
![grafik](https://github.com/Hypercontext/linkifyjs/assets/1484427/9b5d6703-9f8a-4a45-b350-c991e80a56a7)

Can you please interpret this differently for FediVerse addresses?

**Comment by Der-K-2000** [comment]
Yes, as also mentioned [here](https://github.com/nextcloud/tasks/issues/2828) (for the [NextCloud](https://github.com/nextcloud) App [tasks](https://github.com/nextcloud/tasks)). Please fix this!
